stackn/create.py

```diff
@@ -168,6 +169,17 @@ def project(name, description, template, studio_url, secure):
     )
 
 
+@create.command("client")
+@click.argument("name")
+@click.option("-p", "--project", required=False, default=[])
+@click.option("-c", "--controller-id", required=False, default=[])
+@click.option("-r", "--role", required=False, default=[])
+@click.option("-u", "--studio-url", required=False, default=[])
+@click.option("--secure/--insecure", default=True)
+def client(name, project, controller_id, role, studio_url, secure):
+    create_client(name, project, controller_id, role, studio_url, secure)
```

Comment:
function can return False, if the value is not used do: _ = create_client(**args)

Reply:
Updated so that function never returns